modules/sdk/src/main/scala/org/tessellation/sdk/config/types.scala

```diff
@@ -69,4 +70,8 @@ object types {
     defaultCheckAttempts: PosInt,
     ensureCheckInterval: FiniteDuration
   )
+
+  case class CollateralConfig(
+    threshold: Amount
```

Comment:
I'd rename it into `minStaking` or `minStakingAmount`

Reply:
Maybe just `amount` given that it's inside of `CollateralConfig` or `collateral` simply.

Reply:
I ahve chabged it to `minStakingAmount`. @TheMMaciek ,if you are really unhappy with this I can change as you suggested.

Reply:
It is amount now.